app/serializers/person_escort_records_serializer.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class PersonEscortRecordsSerializer
+  include JSONAPI::Serializer
+  include JSONAPI::ConditionalRelationships
+
+  set_type :person_escort_records
+
+  has_many_if_included :flags, serializer: FrameworkFlagsSerializer, &:framework_flags
+
+  attributes :confirmed_at, :created_at, :nomis_sync_status
```

Comment:
do we not need a `SUPPORTED_RELATIONSHIPS` for flags here?

Reply:
No because it doesn't get rendered directly from a controller